src/test/java/dk/kb/present/solr/EmbeddedSolrFieldAnalyseTest.java

```diff
@@ -196,6 +224,31 @@ private static void addTestDocuments() throws Exception {
 
 	}
 
+	
+
+	
+	private static void addSynonymFieldTestDocuments()  {
+
+	try {
+				SolrInputDocument document = new SolrInputDocument();
+				document.addField("id", "synonym1");
+				document.addField("origin", "ds.test");
+				document.addField("title", "Velkommen til TV avisen");  //Synonym file: tv-avisen, tvavis, tvavisen, tv-avis => tv avisen
```

Comment:
Could you add a document and a test that reverses the synonym flow? Where `title` is `Velkommen til TVAvisen` and the search is for `TV Avisen`?

Reply:
Fixed, and the uppercase TVAvisen revealed and error in the fieldtype analyser. lowercase-filer must be before synonyms (and stopwords).

Reply:
I might have been unclear here: My point is that synonym handling of multi-words needs to be checked both ways.

1. A search for `tv avisen` should match a document where `tvavisen` is indexed
2. A search for `tvavisen` should match a document where `tv avisen` is indexed

The unit test needs at least 2 documents, one with and one without space in `tv avisen`.

Reply:
Two different DOCs now and unittest

Reply:
I don't see an unit test for case 1: Search for `tv avisen` (with space in the query).

Reply:
Added search in both document for tv avisen and "tv avisen" (quotes)